pulp_python/app/pypi/views.py

```diff
@@ -124,18 +124,25 @@ def upload(self, request, path):
         if repo_content.filter(filename=filename).exists():
             return HttpResponseBadRequest(reason=f"Package {filename} already exists in index")
 
-        return self.upload_package(repo, artifact, filename, request.session)
+        if settings.GROUP_UPLOADS:
```

Comment:
Is this a feature which many plugins can or might eventually use?

Reply:
I should probably prepend PYTHON to this setting since it's pulp_python specific.

Reply:
I would ask whether other plugins might want to use this strategy - if yes then that might not be necessary.

Reply:
Yeah, but since I set the default for this setting in pulp_python it could conflict with the defaults for other plugins if they were to use it.